### ISSUE TITLE
Client: Describe command

### ISSUE THREAD

**Issue by technosophos** [kubernetes/helm#242]
The `helm describe NAME` will query the DM server for information about a chart named NAME, and then display the results.

**Comment by michelleN** [comment]
Blocked on #367. 
More details at issue #458.

**Comment by michelleN** [comment]
cc/ @vaikas-google

**Comment by vaikas-google** [comment]
Just to clarify, this is going to describe the resulting k8s resources that are part of this deployed chart? Layout is not part of this, but eventually would be as well? Just making sure I got the gist of it :)

**Comment by vaikas-google** [comment]
One more clarifying question:
 1. Should it do a get against all the resources to display their current state? The manifest keeps the state [created/failed/aborted (iirc)], but it seems like it might be of interest to know:
   1.a - It was created successfully (or it failed)
   2.a - What is the resources current state [for example did somebody delete it from underneath]

**Comment by jackgr** [comment]
I'd like to see us provide as much information as possible, including the layout and resource deployment status. Probably good to let the use control the level of verbosity with flag(s).

**Comment by vaikas-google** [comment]
Yeah, was thinking of adding a flag and with it, it would actually talk to k8s. Without it, only display the resources. Good enough, just wanted to get an lgtm on the plan as the original description was a bit vague :)

**Comment by jackgr** [comment]
LGTM

**Comment by technosophos** [comment]
Yeah, I agree. I believe the expectation for this command is that it will produce some verbose output (like `kubectl describe`). It is probably fair to assume that the client may not have access to the k8s API surface. So if there is k8s API code, we should hide it behind a flag, as you suggested. Besides, repeating k8s API structures isn't all that useful if the user can simply run the `kubectl` commands on their own.

**Comment by vaikas-google** [comment]
righteyroo. I'll be working on this...
